nhaystack-rt/src/nhaystack/server/TagManager.java

```diff
@@ -1266,10 +1274,10 @@ private static int getTrendRecordKind(Type trendRecType)
     /** Every single tag which the server may have auto-generated.  */
     static final String[] AUTO_GEN_TAGS = {
         "axAnnotated",
-        "axHistoryId",
+        "n4HistoryId",
```

Comment:
Breaking change.

Reply:
Yep, I've just put these back, we'll generate both so old clients don't get left behind.

Reply:
It's still showing as the axHistoryId as removed, am I reading it wrong?